src/Microsoft.ML.StandardLearners/Standard/MultiClass/doc.xml

```diff
@@ -0,0 +1,59 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<doc>
```

Comment:
:memo: I would greatly prefer to see all these **doc.xml** files appear directly in the source code. If you wanted to define an element once and reuse it, the syntax from SHFB makes sense:

```xml
<inheritdoc cref="ElementToInheritFrom" />
```

Reply:
see the other resp. Short answer: me too:)

---
In reply to: [202775860](https://github.com/dotnet/machinelearning/pull/529#discussion_r202775860) [](ancestors = 202775860)